sos/cleaner/__init__.py

```diff
@@ -0,0 +1,607 @@
+            os.makedirs(os.path.join(self.tmpdir, 'cleaner'), exist_ok=True)
+
+        os.umask(0o77)
+        self.in_place = in_place
+        self.hash_name = self.policy.get_preferred_hash_name()
+
+        self.cleaner_md = self.manifest.components.add_section('cleaner')
+
+        self.parsers = [
+            SoSHostnameParser(self.opts.map_file, self.opts.domains),
+            SoSIPParser(self.opts.map_file),
+            SoSMacParser(self.opts.map_file)
+        ]
+
+        self.log_info("Cleaner initialized. From cmdline: %s"
+                      % self.from_cmdline)
+
+    def _fmt_log_msg(self, msg, caller=None):
+        return "[cleaner%s] %s" % (":%s" % caller if caller else '', msg)
+
+    def log_debug(self, msg, caller=None):
+        self.soslog.debug(self._fmt_log_msg(msg, caller))
+
+    def log_info(self, msg, caller=None):
+        self.soslog.info(self._fmt_log_msg(msg, caller))
+
+    def log_error(self, msg, caller=None):
+        self.soslog.error(self._fmt_log_msg(msg, caller))
+
+    def _fmt_msg(self, msg):
+        width = 80
+        _fmt = ''
+        for line in msg.splitlines():
+            _fmt = _fmt + fill(line, width, replace_whitespace=False) + '\n'
+        return _fmt
+
+    def print_disclaimer(self):
+        """When we are directly running `sos clean`, rather than hooking into
+        SoSCleaner via report or collect, print a disclaimer banner
+        """
+        msg = self._fmt_msg("""\
+This command will attempt to obfuscate information that is generally \
+considered to be potentially sensitive. Such information includes IP \
+addresses, MAC addresses, domain names, and any user-provided keywords.
+
+Note that this utility provides a best-effort approach to data obfuscation, \
+but it does not guarantee that such obfuscation provides complete coverage of \
+all such data in the archive, or that any obfuscation is provided to data that\
+ does not fit the description above.
+
+Users should review any resulting data and/or archives generated or processed \
+by this utility for remaining sensitive content before being passed to a \
+third party.
+""")
+        self.ui_log.info("\nsos clean (version %s)\n" % __version__)
+        self.ui_log.info(msg)
+        if not self.opts.batch:
+            try:
+                input("\nPress ENTER to continue, or CTRL-C to quit.\n")
+            except KeyboardInterrupt:
+                self.ui_log.info("\nExiting on user cancel")
+                self._exit(130)
+
+    @classmethod
+    def add_parser_options(cls, parser):
+        parser.usage = 'sos clean|mask TARGET [options]'
+        clean_grp = parser.add_argument_group(
+            'Cleaner/Masking Options',
+            'These options control how data obfuscation is performed'
+        )
+        clean_grp.add_argument('--domains', action='extend', default=[],
+                               help='List of domain names to obfuscate')
+        clean_grp.add_argument('target',
```

Comment:
Maybe culd we `s/'target'/'TARGET'/g` as we refer to an argument value? See current help:

    # python3 ./bin/sos clean --help
    ..
    Cleaner/Masking Options:
      These options control how data obfuscation is performed
    
      --domains DOMAINS     List of domain names to obfuscate
      target                The directory or archive to obfuscate
      -j JOBS, --jobs JOBS  Number of concurrent archives to clean
      --map MAP_FILE        Provide a previously generated mapping file for
                            obfuscation
      --no-update           Do not update the --map file with new mappings from
                            this run
    #

where `target` is almost invisible.